wandb/sdk/wandb_login.py

```diff
@@ -259,7 +259,8 @@ def _verify_login(self, key: str) -> None:
 
         if not is_api_key_valid:
             raise AuthenticationError(
-                "API key verification failed. Make sure your API key is valid."
+                f"API key verification failed for host {self._settings.base_url}. "
+                "Make sure your API key is valid."
```

Comment:
nit: it's a lot less error-prone to put separating whitespace at the *start* of the continuation string rather than at the end of the previous string.

There's a bug in this string:

```python
"My verbose string of very long length "
"with a lot of unnecessary text to "
"prove a point about"
"continuations"
```

The vertical alignment of spaces makes it a lot easier to spot here:

```python
"My verbose string of very long length"
" with a lot of unnecessary text to"
" prove a point about"
"continuations"
```

Strings are usually not aligned vertically at the end, but are always aligned at the start.